packages/cfd/src/Containers/mt5-migration-modal/mt5-migration-modal.tsx

```diff
@@ -31,7 +31,7 @@ const MT5MigrationModal = observer(() => {
     React.useEffect(() => {
         if (is_mt5_migration_modal_open) {
             const has_mt5_migration_error = !!mt5_migration_error;
-            setShowModalFrontSide(!has_mt5_migration_error);
+            setShowModalFrontSide(has_mt5_migration_error);
```

Comment:
since we only have only front side for the modal,  do we need this ?

Reply:
let me try remove it and see 👌

Reply:
Let me check on this 🙏